gis/openstreetmap-carto/Portfile

```diff
@@ -0,0 +1,210 @@
+categories          gis
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+
+# license CC0-1.0 (http://creativecommons.org/publicdomain/zero/1.0/)
+license             public-domain
+
+description         A general-purpose OpenStreetMap mapnik style, in CartoCSS
+
+long_description    These are the CartoCSS map stylesheets for the Standard \
+                    map layer on OpenStreetMap.org.
+
+checksums           rmd160  b7efb038f0bf942ff8403f72fa34fb2040fa6fdb \
+                    sha256  fd8f398a2a674a84e070e9eacaa4c0cb7de036950f7434e8c259e6b7974b47df \
+                    size    2719400
+
+python.versions     36 37 38 39
+
+python.default_version 39
+
+depends_lib         port:py-requests \
+                    port:py-yaml \
+                    port:py-psycopg2
+
+# Database variants (from the GDAL port)
+set postgresql_suffixes {12 13}
+
+set portsgresql_variants {}
+set postgresql_default_variant "if {"
+
+foreach s ${postgresql_suffixes} {
+    lappend portsgresql_variants postgresql${s}
+    set postgresql_default_variant "${postgresql_default_variant}!\[variant_isset postgresql${s}] && "
+}
+
+set postgresql_default_variant [string range ${postgresql_default_variant} 0 end-4]
+set postgresql_default_variant "${postgresql_default_variant}} {default_variants +postgresql${s}}"
+eval $postgresql_default_variant
+
+foreach s ${postgresql_suffixes} {
+    set p postgresql${s}
+    set v [string index ${s} 0].[string index ${s} 1]
+    set i [lsearch -exact ${portsgresql_variants} ${p}]
+    set c [lreplace ${portsgresql_variants} ${i} ${i}]
+    variant ${p} description "Enable PostgreSQL ${v} support" conflicts {*}${c} "
+        depends_lib-append      port:${p}
+    "
+}
+
+set datadir         ${prefix}/var/lib/${name}
+set configdir       ${prefix}/etc/${name}
+
+use_configure       no
+
+# The `scripts-get-external-data-py.diff` patch has been merged
+# into the upstream master at
+# https://github.com/gravitystorm/openstreetmap-carto/commit/4ec2dc9391c411e124c78b3ba1aad9173fea20cb
+# so it should be possible to remove it after the next release (> v5.3.1).
+patchfiles          style-fonts-mss.diff \
+                    scripts-get-external-data-py.diff
+
+variant postgresql12 conflicts postgresql13 description {Use with PostgreSQL 12} {
+    require_active_variants py36-psycopg2 postgresql12
+    require_active_variants py37-psycopg2 postgresql12
+    require_active_variants py38-psycopg2 postgresql12
+    require_active_variants py39-psycopg2 postgresql12
+}
+
+variant postgresql13 conflicts postgresql12 description {Use with PostgreSQL 13} {
+    require_active_variants py36-psycopg2 postgresql13
+    require_active_variants py37-psycopg2 postgresql13
+    require_active_variants py38-psycopg2 postgresql13
+    require_active_variants py39-psycopg2 postgresql13
+}
+
+post-patch {
+    set python_bin ${prefix}/bin/python3
+    reinplace "s#/usr/bin/env python3#${python_bin}#g" \
```

Comment:
these things are all done by the python PG. You can just use ${python.bin} here

Reply:
Thanks. Done.